community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tests/test_resume.py

```diff
@@ -134,29 +134,29 @@ def mock_tpu_se(ns: TstNodeset) -> TstTPU:
 
 
 @pytest.mark.parametrize(
-    "nodes,excl_job_id,expected",
+    "nodes,max_distance,excl_job_id,expected",
```

Comment:
Instead of making it a test parameter , embed it into nodeset.

Reply:
Done